decred/decred/util/helpers.py

```diff
@@ -128,39 +138,40 @@ def prepareLogging(filepath=None, logLvl=logging.INFO, lvlMap=None):
         LogSettings.root.addHandler(printHandler)
 
 
-def getLogger(name):
+def getLogger(name: str) -> Logger:
     """
     Gets a named logger. If the name has a log level registered with
     prepareLogger, that level will be used, otherwise the default is used.
 
     Args:
-        name (str): The logger name.
+        name: The logger name.
     """
     l = LogSettings.root.getChild(name)
     l.setLevel(LogSettings.moduleLevels.get(name, LogSettings.defaultLevel))
     LogSettings.loggers[name] = l
     return l
 
 
-def readINI(path, keys):
+def readINI(path: str, keys: Iterable[str]) -> Optional[Dict[str, str]]:
```

Comment:
What is the use of `Optional` for the return value here?

Reply:
It was added automatically but it's not useful because the function always returns a Dict, removing it.